Docs - date_sub (#20163)
ClickHouse/ClickHouse: docs/en/sql-reference/functions/date-time-functions.md

**Parameters**

> -   `unit` — The type of interval to truncate the result. [String Literal](../syntax.md#syntax-string-literal).

Author: Minor unrelated change which I overlooked this one in my last PR. This is actually string literal as the `unit` is specified as `'hour'` , `'minute'`, `'second'` etc.

Example:
```sql
ELECT now(), date_trunc('hour', now(), 'Europe/Moscow')
```

└────────────────────────────────────────────────────────────────────────────────────────┘
```

## date\_sub {#date_sub}

> This subtracts a time/date interval from a date and then returns the date.

Reviewer: ```suggestion
Subtracts a time/date interval from the provided date.
```

**Syntax**

``` sql
date_sub(unit, value, date)
```

Aliases: `dateSub`, `DATE_SUB`. 

**Parameters**

-   `unit` — The type of interval to subtract. [String](../../sql-reference/data-types/string.md).

        Supported values: second, minute, hour, day, week, month, quarter, year.
-   `value` - Value in specified unit - [Int](../../sql-reference/data-types/int-uint.md)    

> -   `date` — [Date](../../sql-reference/data-types/date.md) or [DateTime](../../sql-reference/data-types/datetime.md).

Reviewer: ```suggestion
-   `date` — [Date](../../sql-reference/data-types/date.md) or [DateTime](../../sql-reference/data-types/datetime.md) to subtract value from.
```
